Makefile

```diff
@@ -85,7 +85,8 @@ cvl-test:
 
 rest-server: translib
 	$(MAKE) -C src/rest
-
+rest:
+	$(MAKE) -C src/rest
```

Comment:
Why this rule is needed ?

Reply:
Sorry partha, added just for easy compilation keyword. Fixed.